app/helpers/notification_subscriptions_helper.rb

```diff
@@ -1,8 +1,8 @@
 module NotificationSubscriptionsHelper
-  def render_notification_subscription_component(subscription_types, journey: nil)
+  def render_notification_subscription_component(subscription_types: {}, journey: nil, extra_attributes: {})
```

Comment:
Taketo keyword argumenty su v Ruby/Rails nestandardne (co je v Rails problematicke, kedze tu razime convention over configuration). Navrhujem revertnut https://github.com/slovensko-digital/navody.digital/pull/255/commits/415eb2a068c5bf1c5a38c19d2f191f9f6e20b7c7 a spravit tak ako som pisal [vyssie](https://github.com/slovensko-digital/navody.digital/pull/255#discussion_r322025490):
```suggestion
  def render_notification_subscription_component(subscription_types, journey = nil, extra_attributes = {})
```

Reply:
@eronisko  tam mam ale problem ze ak nechcem zakazdim zadavat `journey: nil` argument tak mi bude optional `journey` brat mylne `extra_attributes` v invokacii `render_notification_subscription_component(subscription_types, extra_attributes)`

definicia `def render_notification_subscription_component(subscription_types, extra_attributes = {},  journey = nil)` moze byt?